netplugin/netd.go

```diff
@@ -43,9 +67,9 @@ type cliOpts struct {
 	debug      bool
 	syslog     string
 	jsonLog    bool
-	ctrlIP     string // IP address to be used by control protocols
-	vtepIP     string // IP address to be used by the VTEP
-	vlanIntf   string // Uplink interface for VLAN switching
+	ctrlIP     string      // IP address to be used by control protocols
+	vtepIP     string      // IP address to be used by the VTEP
+	vlanIntf   StringSlice // Uplink interface for VLAN switching
```

Comment:
Yes, I've made changes to StringSlice Set function to handle comma separated list as well